platforms/guardoni/src/guardoni/utils.ts

```diff
@@ -18,6 +18,7 @@ export const CHROME_PATHS = [
   'C:\\Program Files (x86)\\Google\\Chrome\\Application\\chrome.exe',
   'C:\\Program Files\\Google\\Chrome\\Application\\chrome.exe',
   '/Applications/Chromium.app/Contents/MacOS/Chromium',
+  '/Applications/Google Chrome.app/Contents/MacOS/Google Chrome',
```

Comment:
this is important, i really missed it, and it was very wrong. i wonder how and if was working on MacOS in the same way as the other. thanks